pkg/boot/universalpayload/universalpayload_test.go

```diff
@@ -64,7 +64,12 @@ address sizes	: 39 bits physical, 48 bits virtual
 	defer os.Remove(tempFile)
 	// end of mock data
 
-	mem, loadErr := loadKexecMemWithHOBs(fdtLoad, data)
+	ioMem, err := kexecMemoryMapFromIOMem()
+	mem := kexec.Memory{
+		Phys: ioMem,
+	}
+
+	_, loadErr := loadKexecMemWithHOBs(fdtLoad, data, &mem)
 	if loadErr != nil {
 		t.Fatalf("Unexpected err: %+v", loadErr)
```

Comment:
nit: simplify error checking when the result is not used (`_`)
```suggestion
	if loadErr := loadKexecMemWithHOBs(fdtLoad, data, &mem); loeadErr != nil {
		t.Fatalf("Unexpected err: %+v", loadErr)
```